bigquery/spec/bigquery_sample_spec.rb

```diff
@@ -0,0 +1,336 @@
+# Copyright 2016 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "../datasets"
+require_relative "../tables"
+require "rspec"
+require "google/cloud"
+require "csv"
+
+RSpec.describe "Google Cloud BigQuery samples" do
+
```

Comment:
There's no hard and fast rule for it.

I usually put a space after my `describe` block, otherwise the example description is squished and doesn't stand out.  You'll see both if you browse code samples.

#fyi I specifically set a RuboCop rule to allow for this [[1](https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/master/.rubocop.yml#L75)]

My personal rule has always been: *always add a newline before an example*

I don't always add the newline before `before` blocks, though, so the samples are inconsistent 😕 

Putting together a style guide is a high priority, see also: #51